### PR TITLE
Clarify fragment, fix deadlink

### PR DESCRIPTION
**Pull request by DaedalusG**
This is just a quick PR to fix up a fragment and a link that was keyed to localhost

### CODE REVIEW

File: doc/code_intelligence/explanations/uploads.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Administrators of a Sourcegraph instance can see a global view of precise code i
 
 Sourcegraph keeps a mapping from a commit of a repository to the set of upload records that can resolve a query for that commit. When an upload record moves into or away from the `COMPLETED` state, the set of eligible uploads change and this mapping must be recalculated.
 
-When an upload changes state, we flag the repository as needing to be updated. Then the [`worker` service](http://localhost:5080/admin/workers#codeintel-commitgraph)
+When an upload changes state, we flag the repository as needing to be updated. Then the [`worker` service](https://docs.sourcegraph.com/admin/workers#codeintel-commitgraph)
```
Comment (efritz) [inline doc/code_intelligence/explanations/uploads.md:51]: 🤦

**Review by efritz — APPROVED** [review]

File: doc/code_intelligence/explanations/uploads.md (hunk 1 above)
Comment (DaedalusG) [reply doc/code_intelligence/explanations/uploads.md:51]: Actually I was about to ask, where is this computationally expensive? Is it for the gitserver?

File: doc/code_intelligence/explanations/uploads.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Processing of a precise code intelligence index file may occur due to malformed
 
 At any point, the upload record may be deleted. This can happen because the record is being replaced by a newer upload, due to [age of the upload record](../how-to/configure_data_retention.md), or due to explicit deletion by the user. Deleting a record that could be used to resolve to code intelligence queries will first move into the `DELETING` state. Moving temporarily into this state allows Sourcegraph to smoothly transition the set of code intelligence uploads that are visible for query resolution.
 
-The change of an upload into or away from the `COMPLETED` state can be an expensive one, and when is the [repository commit graph](#repository-commit-graph) needs to be refreshed.
+Changing the state of an upload to or from the `COMPLETED` state can be computationally expensive. When such a change is made, the [repository commit graph](#repository-commit-graph) needs to be refreshed.
```
Comment (DaedalusG) [inline doc/code_intelligence/explanations/uploads.md:32]: @efritz I was hoping to clarify this a little further. What was the intention of this bit, which services is the computational load on? Gitserver?

Comment (efritz) [reply doc/code_intelligence/explanations/uploads.md:32]: `worker` service and/or `postgres` database: https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/enterprise/cmd/worker/internal/codeintel/commitgraph/updater.go?L105

Comment (DaedalusG) [reply doc/code_intelligence/explanations/uploads.md]: Updated https://github.com/sourcegraph/sourcegraph/pull/28743/commits/87443aa332cf846ca59d54ed0595ef2bfec1822d

**Pull request merged**